### DISCUSSION TITLE
The select drop-down box has no options

### DISCUSSION THREAD

**Discussion by zhaokaixin** [apache/camel-karavan#879] [Q&A] [answered]
![image](https://github.com/apache/camel-karavan/assets/22127268/9b892499-3031-40c2-aa38-433824a26244)
![image](https://github.com/apache/camel-karavan/assets/22127268/f5665bba-85d2-41c8-88f4-d95f6767c751)
I found that there are no options in some drop-down boxes, how to set this?

**Comment by davsclaus** [comment, marked as answer]
You need to create a "bean" first

**Comment by zhaokaixin** [comment]
thank you.Is there a detailed operating manual?

**Reply by davsclaus** [reply]
No there is no manual
